.github/workflows/kvrocks.yaml

```diff
@@ -160,6 +160,27 @@ jobs:
           export ${{ matrix.runtime_env_vars }}
           ./build/unittest
 
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+
+      - name: Run Go Integration Cases
+        working-directory: tests/gocase
+        run: |
+          export ${{ matrix.runtime_env_vars }}
+          export KVROCKS_BIN_PATH="$GITHUB_WORKSPACE/build/kvrocks"
+          export GO_CASE_WORKSPACE="$GITHUB_WORKSPACE/tests/gocase/workspace"
+          go test -v ./...
```

Comment:
Could these be added in `./x.py`, like `./x.py test go <build-dir>`?

Reply:
Yes, it's possible. I'd like to do it as a follow-up, though.

Reply:
got it